src/main/java/use_case/translateText/TranslateTextInputBoundary.java

```diff
@@ -0,0 +1,13 @@
+package use_case.translateText;
+
+/**
+ * Input Boundary for actions which are related to translating text.
+ */
+public interface TranslateTextInputBoundary {
+
+    /**
+     * Executes the translatetext use case.
+     */
+    void execute();
```

Comment:
the method signature for execute here should match the controller's execute() call (ie. you should have input text, input language, and output language